keyboards/gowla/keymaps/default/keymap.c

```diff
@@ -0,0 +1,29 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[0] = LAYOUT( \
+  LT(1, KC_MPRV),       KC_MNXT,      	KC_MPLY, \
+  KC_VOLD,       		KC_UP,			KC_VOLU, \
+  KC_LEFT,       		KC_DOWN,      	KC_RIGHT \
+),
+
+[1] = LAYOUT( \
+  KC_ESC,   KC_MPLY, 	RESET, \
+  KC_P7, 	KC_P1, 		M(0), \
+  KC_F1,	KC_F2,  	M(1)  \
+)
+
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
`action_get_macro()` is deprecated: https://docs.qmk.fm/#/feature_macros?id=the-new-way-send_string-amp-process_record_user